Design Patterns/Abstract Factory Pattern/src/GPUs/GpuBrand.java

```diff
@@ -0,0 +1,5 @@
+package GPUs;
+
+public interface GpuBrand extends GpuModel {
```

Comment:
please add documentation here, and mention the needs for interface.

Reply:
apply it for all

Reply:
should be done by now.